model/types.go

```diff
@@ -19,43 +19,52 @@ import (
 	"time"
 
 	"k8s.io/heapster/sinks/cache"
-	"k8s.io/heapster/store"
+	"k8s.io/heapster/store/daystore"
+	"k8s.io/heapster/store/statstore"
 )
 
 type Cluster interface {
 	// The Update operation populates the Cluster from a cache.
 	Update(cache.Cache) error
 
+	// The simple Get operations extract structural information from the Cluster.
+	GetAvailableMetrics() []string
+	GetNodes() []EntityListEntry
+	GetNamespaces() []EntityListEntry
```

Comment:
the interface can be renamed to Model to reflect its holistic scope of the cluster's structure. Having sub-objects would require a redesign of the current locking mechanisms, as we have discussed in the past